R/run.R

```diff
@@ -0,0 +1,102 @@
+run <- function(pathmaxent="/home/affu/Desktop/Maxent/", pathproject="/home/affu/Desktop/example/", doproject=0,
```

Comment:
follow [this](http://adv-r.had.co.nz/Style.html) for spaces and indentation -- not a must though